### ISSUE TITLE
🐛 Selection not working properly on very large tables

### ISSUE THREAD

**Issue by raymeskhoury** [inovua/reactdatagrid#295]
Hi there, 

This component looks great. I'm considering using and purchasing React Data Grid for a commercial project however have encountered a bug that would stop us below. Thanks for taking a look!

- what edition are you using - same issue exists in both

Relevant code or config

```javascript
import React from 'react'

import ReactDataGrid from '@inovua/reactdatagrid-enterprise'
import '@inovua/reactdatagrid-enterprise/index.css'

const gridStyle = { minHeight: 550 }

const columns = [
  { name: 'id', header: 'Id', defaultWidth: 80, defaultVisible: false },
  { name: 'name', sortable: false, header: 'Name (column not sortable)', defaultFlex: 1 },
  { name: 'age', header: 'Age', type: 'number', defaultFlex: 1 }
];

let dataSource = [];

for (let i = 0; i < 50000; ++i) {
  dataSource.push({ name: "Little Johny", age: i, id: i });
}

const App = () => {
  return (
    <ReactDataGrid
      idProperty="id"
      style={gridStyle}
      columns={columns}
      dataSource={dataSource}
    />
  )
}

export default () => <App />
```

What you did/What happened:: Scroll to the last row and click it, notice that the row doesn't get properly selected or highlighted. The row gets highlighted on hover, but no selection border when clicked.

Rows at the start of the table (row number 25,000 or less) seem to get selected with no problems, however above that and selection suddenly stops working.

Reproduction repository: I just slightly modified the example at https://reactdatagrid.io/docs/sorting using the code above. The above code can be pasted in.

Problem description: Selection and navigation don't work for any rows past about 25,000.

Suggested solution: Not sure where the bug lies.

**Comment by inovua-admin** [comment]
Fixed in v `5.0.0-canary.1` - stable version `5.0.0` coming soon

**Comment by inovua-admin** [comment]
Fixed in version `5.0.0`.
